packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -8140,21 +8140,89 @@ interface IGroupedListStyles {
   root: IStyle;
 }
 
+// @public (undocumented)
+interface IGroupFooterProps extends IGroupDividerProps {
+  className?: string;

```

Comment:
Should className and theme go in the base IGroupDividerProps since they are common to both header and footer props?